lib/ssl/src/dtls_connection.erl

```diff
@@ -734,13 +734,14 @@ handle_client_hello(#client_hello{client_version = ClientVersion} = Hello, State
 
 handle_state_timeout(flight_retransmission_timeout, StateName,
                      #state{protocol_specific = 
-                                #{flight_state := {retransmit, _NextTimeout}}} = State0) ->
+                                #{flight_state := {retransmit, CurrentTimeout}}} = State0) ->
     {State1, Actions0} = dtls_gen_connection:send_handshake_flight(State0, 
                                                                    retransmit_epoch(StateName, State0)),
-    {next_state, StateName, State, Actions} = 
+    {next_state, StateName, #state{protocol_specific = PS} = State, Actions} =
         dtls_gen_connection:next_event(StateName, no_record, State1, Actions0),
+    State2 = State#state{protocol_specific = PS#{flight_state => {retransmit, new_timeout(CurrentTimeout)}}},
     %% This will reset the retransmission timer by repeating the enter state event
-    {repeat_state, State, Actions}.
+    {repeat_state, State2, Actions}.
```

Comment:
We have a convention to number the states and call the final one only State, so I would switch State2 and State!

Reply:
Done